services.go

```diff
@@ -767,6 +767,103 @@ func (s *ServicesService) DeleteGithubService(pid interface{}, options ...Reques
 	return s.client.Do(req, nil)
 }
 
+// HarborService represents the Harbor service settings.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/integrations.html#harbor
+type HarborService struct {
+	Service
+	Properties *HarborServiceProperties `json:"properties"`
+}
+
+// HarborServiceProperties represents Harbor specific properties.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/integrations.html#harbor
+type HarborServiceProperties struct {
+	URL                  string    `json:"url"`
+	ProjectName          string    `json:"project_name"`
+	Username             string    `json:"username"`
+	Password             string    `json:"password"`
+	UseInheritedSettings BoolValue `json:"use_inherited_settings"`
```

Comment:
As far as I can tell this should be a regular `bool`

Reply:
Updated! Thank you for the quick feedback.